pydm/tests/utilities/test_utilities.py

```diff
@@ -51,6 +51,27 @@ def test_find_display_in_path():
     assert disp_path == expected
 
 
+def test_find_file():
+    parent_lvl1 = tempfile.mkdtemp()
+    parent_lvl2 = tempfile.mkdtemp(dir=parent_lvl1)
+    temp, file_path = tempfile.mkstemp(suffix=".ui", prefix="display_", dir=parent_lvl2)
+    direc, fname, _ = path_info(file_path)
+    # Try to find the file as is... is should not find it.
```

Comment:
```suggestion
    # Try to find the file as is... it should not find it.
```

Reply:
Copied this comment from a previously defined function :) Fixed